yucca/training/augmentation/YuccaAugmentationComposer.py

```diff
@@ -213,6 +215,7 @@ def compose_train_transforms(self):
                 DownsampleSegForDS(deep_supervision=self.deep_supervision),
                 CopyImageToSeg(copy=self.copy_image_to_label),
                 Masking(mask=self.mask_image_for_reconstruction, pixel_value=self.cval, ratio=self.mask_ratio),
+                Normalize(normalize=self.normalize),
```

Comment:
so this:
```
DownsampleSegForDS(deep_supervision=self.deep_supervision),
CopyImageToSeg(copy=self.copy_image_to_label),
Masking(mask=self.mask_image_for_reconstruction, pixel_value=self.cval, ratio=self.mask_ratio),
Normalize(normalize=self.normalize),
```
becomes this:
```
DownsampleSegForDS(deep_supervision=self.deep_supervision),
Normalize(normalize=self.normalize),
CopyImageToSeg(copy=self.copy_image_to_label),
Masking(mask=self.mask_image_for_reconstruction, pixel_value=self.cval, ratio=self.mask_ratio),
```

Reply:
Ah I see you already fixed this, figured it was nicer if I added it in the review

Reply:
I moved it up one more. Is that ok or should i change?

Reply:
Nono, thats fine it makes no difference